resources/public/api/conf/1.0/application.yaml

```
@@ -30,6 +30,17 @@ components:
           scopes:
             write:self-assessment: Grant write access
             read:self-assessment: Grant read access
+    Application-Restricted:
+      type: oauth2
+      description: |
+        HMRC supports OAuth 2.0 for authenticating application restricted API requests using an OAuth 2.0 Bearer Token
+        in the AUTHORIZATION header. See https://developer.service.hmrc.gov.uk/api-documentation/docs/authorisation/application-restricted-endpoints for details.
```

I think it's more consistent with the rest of our OAS docs that we hide the raw url inside some linked text. Perhaps something like this, which follows the format of the user-restricted description? 
```suggestion
        HMRC supports OAuth 2.0 for authenticating [Application-restricted](https://developer.service.hmrc.gov.uk/api-documentation/docs/authorisation/application-restricted-endpoints) API requests.
```